http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/DiskRoutes.scala

```diff
@@ -57,7 +55,7 @@ class DiskRoutes(diskService: DiskService[IO], userInfoDirectives: UserInfoDirec
               }
             } ~
               pathPrefix(Segment) { diskNameString =>
-                validateDiskNameDirective(diskNameString) { diskName =>
+                LeoRoutes.validateNameDirective(diskNameString, DiskName.apply) { diskName =>
```

Comment:
Yeah... I held off because the object this is contained in is an overloaded name, which makes intellij unhappy with refactoring... I'll bite the bullet

Reply:
thanks 🙏